docs/cloud-guidance/azure/ukho-azure-tagging.md

```diff
@@ -0,0 +1,28 @@
+# UKHO Azure Tagging Strategy v0.1 DRAFT
+
+## How to use this document
+
+This document outlines which tags should be applied when creating resources in the cloud. Use the table below to ensure you are conforming to the UKHO tagging strategy.
+
+## How to contribute to this document
+
+The owner of this document is Digital Operations. Please submit suggestions/changes/amendments vie a pull request and inform Digital Operations.
+
+## References
+
+The following has been used to inform the creation of these standards:
+
+- [Microsoft Using Tags to Organise Azure Resources](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-using-tags)
+- [AWS Tagging Strategies](https://aws.amazon.com/answers/account-management/aws-tagging-strategies/)
+- [Google Labelling Resources](https://cloud.google.com/compute/docs/labeling-resources)
+- UKHO Solarwinds
+
+## Tags
```

Comment:
Move this above `How to contribute` and `References`. Don't make me scroll. People don't really care who wrote it, just what it says.

Reply:
Done